src/player.cpp

```diff
@@ -251,42 +252,38 @@ void Player::Update(bool update_scene) {
 	// Ticks in emscripten are unreliable due to how the main loop works:
 	// This function is only called 60 times per second instead of theoretical
 	// 1000s of times.
-	Graphics::Update(true);
+	Graphics::Draw();
 #else
-	// Time left before next frame? Let's render the current frame.
 	double cur_time = (double)DisplayUi->GetTicks();
 	if (cur_time < next_frame) {
-		Graphics::Update(true);
-
+		Graphics::Draw();
 		cur_time = (double)DisplayUi->GetTicks();
 		// Still time after graphic update? Yield until it's time for next one.
 		if (cur_time < next_frame) {
 			DisplayUi->Sleep((uint32_t)(next_frame - cur_time));
 		}
-	} else {
-		Graphics::Update(false);
 	}
 #endif
 
-	// Normal logic update
+	// Input:
 	if (Input::IsTriggered(Input::TOGGLE_FPS)) {
 		fps_flag = !fps_flag;
-	}
+		}
 	if (Input::IsTriggered(Input::TAKE_SCREENSHOT)) {
 		Output::TakeScreenshot();
-	}
+		}
 	if (Input::IsTriggered(Input::SHOW_LOG)) {
 		Output::ToggleLog();
-	}
+		}
 	if (Input::IsTriggered(Input::RESET)) {
 		reset_flag = true;
-	}
+		}
 	if (Input::IsTriggered(Input::TOGGLE_ZOOM)) {
 		DisplayUi->ToggleZoom();
-	}
+		}
 	if (Input::IsTriggered(Input::TOGGLE_FULLSCREEN)) {
 		DisplayUi->ToggleFullscreen();
-	}
+		}
```

Comment:
Not sure if github renders this incorrectly but the new indentation looks wrong.